Automattic/vip-go-mu-plugins
# Adding Akismet autoconnection

This PR extends Jetpack Connection Pilot capability to add autoconnection to Akismet. If a Jetpack connection is active, the system will try to connect automatically to Akismet

## Description

This PR extends Jetpack Connection Pilot capability to add autoconnection to Akismet. If a Jetpack connection is active, the system will try to connect automatically to Akismet.

## Changelog Description

### Akismet auto connection

Connection Pilot is now capable of autoconnecting to Akismet.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Go to a Sandbox and launch a CLI.
3. Run the `reconnect()` routine from Connection Pilot.
